articles/ros2_threat_model.md

```diff
@@ -3300,104 +3550,1516 @@ Over time, new reference platforms may be introduced in addition to
+  * `SpecsRotaryServo.srv` is a HRIM message which reports the main features of the device.
+  * `EnableDisable.srv` disables or enables the servo motor.
+  * `ClearFault.srv` sends a request to clear any fault in the servo motor.
+  * `Stop.srv` requests to stop any ongoing movement.
+  * `OpenCloseBrake.srv` opens or closes the servo motor brake.
+
+* **Actions**:
+  * `GoalJointTrajectory` allows to move the joint using a trajectory msg.
+
+Such API gets translated into the following abstractions:
+
+| Topic | Name                                                |
+| ----- | --------------------------------------------------- |
+| goal  | /hrim_actuation_servomotor_XXXXXXXXXXXX/goal_axis1  |
+| goal  | /hrim_actuation_servomotor_XXXXXXXXXXXX/goal_axis2  |
+| state | /hrim_actuation_servomotor_XXXXXXXXXXXX/state_axis1 |
+| state | /hrim_actuation_servomotor_XXXXXXXXXXXX/state_axis2 |
+
+<br>
+
+| Service       | Name                                                      |
+| ------------- | --------------------------------------------------------- |
+| specs         | /hrim_actuation_servomotor_XXXXXXXXXXXX/specs             |
+| enable servo  | /hrim_actuation_servomotor_XXXXXXXXXXXX/enable            |
+| disable servo | /hrim_actuation_servomotor_XXXXXXXXXXXX/disable           |
+| clear fault   | /hrim_actuation_servomotor_XXXXXXXXXXXX/clear_fault       |
+| stop          | /hrim_actuation_servomotor_XXXXXXXXXXXX/stop_axis1        |
+| stop          | /hrim_actuation_servomotor_XXXXXXXXXXXX/stop_axis2        |
+| close brake   | /hrim_actuation_servomotor_XXXXXXXXXXXX/close_brake_axis1 |
+| close brake   | /hrim_actuation_servomotor_XXXXXXXXXXXX/close_brake_axis2 |
+| open brake    | /hrim_actuation_servomotor_XXXXXXXXXXXX/open_brake_axis1  |
+| open brake    | /hrim_actuation_servomotor_XXXXXXXXXXXX/open_brake_axis2  |
+
+<br>
+
+| Action                | Name                                                     |
+| --------------------- | -------------------------------------------------------- |
+| goal joint trajectory | /hrim_actuation_servomotor_XXXXXXXXXXXX/trajectory_axis1 |
+| goal joint trajectory | /hrim_actuation_servomotor_XXXXXXXXXXXX/trajectory_axis2 |
+
+<br>
+
+| Parameters            | Name                                                     |
+| --------------------- | -------------------------------------------------------- |
+| ecat_interface | Ethercat interface |
+| reduction_ratio | Factor to calculate the position of the motor |
+| position_factor | Factor to calculate the position of the motor |
+| torque_factor | Factor to calculate the torque of the motor |
+| count_zeros_axis1 | Axis 1 absolute value of the encoder for the zero position |
+| count_zeros_axis2 | Axis 2 absolute value of the encoder for the zero position |
+| enable_logging| Enable/Disable logging |
+| axis1_min_position | Axis 1 minimun position in radians |
```

Comment:
minimun -> minimum
maximun -> maximum
(here and below)

Reply:
Fixed.